files/image_config/memory_statistics/memory_statistics.conf

```diff
@@ -0,0 +1,10 @@
+# Memory Statistics Daemon Configuration
```

Comment:
Would it better if these configurations can put into config db instead of a static conf file?

Reply:
We have stored configurations in both config_db and a static conf file for flexibility and reliability. 

At startup, the daemon reads default settings from the conf file to ensure a consistent state after a restart. During runtime, the daemon dynamically updates its configuration from config_db without needing a restart. This approach combines the stability of predefined defaults with the convenience of live updates, ensuring predictable behavior and seamless adaptability.